release-notes/opensearch-anomaly-detection.release-notes-1.3.3.0.md

```diff
@@ -7,3 +7,6 @@ Compatible with OpenSearch 1.3.3
 * Use current time as training data end time  ([#547](https://github.com/opensearch-project/anomaly-detection/pull/547))
 * Support writing features using filter aggregation ([#425](https://github.com/opensearch-project/anomaly-detection/pull/425))
 
+### Infrastructure
+* removed bwc zips and fetching opendistro from cloudfront ([#565](https://github.com/opensearch-project/anomaly-detection/pull/565))
```

Comment:
I see usually we have a one line space between title and bullet point, might cause issue for core release-notes parse

Reply:
just fixed.